css/main.css

```diff
@@ -325,3 +325,133 @@ li {
 .skills {
   list-style: inside url("../media/svg/circle.svg");
 }
+
+#contact_form {
+  display: flex;
+  flex-direction: column;
+  width: 375px;
+  margin: auto;
+  background: #fff;
+  height: 100%;
+}
+
+#contact_form p {
+  margin: 120px 24px 73px 24px;
+  text-align: center;
+  font-weight: 700;
+  font-size: 32px;
+  font-family: "Inter", sans-serif;
+  color: #172b4d;
+  line-height: 44px;
+}
+
+#contact_form p span {
+  display: none;
+}
+
+form {
+  margin-left: 24px;
+  margin-right: 24px;
+  display: flex;
+  flex-direction: column;
+}
+
+form input {
+  padding: 10px;
+  border: none;
+  outline: none;
+  background: #fdf7f7;
+  border-bottom: 1px solid #dbd8d7;
+  color: #979493;
+}
+
+textarea {
+  border: none;
+  background: #fdf7f7;
+  color: #3c3a39;
+  font-family: "Inter", sans-serif;
+  font-size: 15px;
+  line-height: 24px;
+}
+
+#contact_form button:enabled {
+  background-color: #ff6b00;
+}
+
+#contact_form button:hover {
+  background-color: #fff5e1;
+  transform: scale(1.1);
+  color: #e05304;
+  border: 1px solid #e05304;
+}
+
+#contact_form button:active {
+  background-color: #e05304;
+  color: #f4f1f0;
+}
+
+#contact_form button:disabled {
+  background-color: #f4f1f0;
+  color: #5b5957;
+  border: 1px solid #5b5957;
+}
```

Comment:
- [ ] As the previous reviewer asked, It would be perfect if you restyle your button and implement the interactions to be like the original Figma design.